.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Go
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15
```

Comment:
1.16 is available - I know it contains some security fixes over 1.15 so might be worth rev'ing to that.

Reply:
Make sense

Reply:
Done, and fixed unit tests